### ISSUE TITLE
Add reset functionality for Mega Drive/Genesis

### ISSUE THREAD

**Issue by caleb39411** [petrockblog/ControlBlockService2#30]
Can reset functionality be added for Mega Drive? There are spare pins, so I don't see why this feature cannot be added in future.

**Comment by petrockblog** [comment]
True, there are spare pins for that. I try to add that during the next week. Any pull request is welcome, of course :-)

**Comment by caleb39411** [comment]
I have almost no experience with this sort of thing, so I'll be of no use, sorry! :smiley:
